deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -266,15 +274,20 @@ class ChatViewController: UITableViewController {
         configureEmptyStateView()
 
         if !disableWriting {
-            configureMessageInputBar()
-            draft.parse(draftMsg: dcContext.getDraft(chatId: chatId))
-            messageInputBar.inputTextView.text = draft.text
-            configureDraftArea(draft: draft, animated: false)
-            editingBar.delegate = self
```

Comment:
`editingBar.delegate = self`  is not needed in `configureUIForWriting()` as it was always already done in `lazy var editingBar`, right?

Reply:
that's right